README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.com/Yelp/detect-secrets.svg?branch=master)](https://travis-ci.com/Yelp/detect-secrets)
+[![Build Status](https://github.com/Yelp/detect-secrets/actions/workflows/ci.yml/badge.svg)](https://github.com/Yelp/detect-secrets/actions/workflows/ci.yml?query=branch%3Amaster++)
```

Comment:
You might want to switch these (use `?branch=master` for the image and let the link show all runs), otherwise I think CI runs on PRs will affect the badge?

Reply:
The image will use the default branch of the repo if not specified. The link should just link to master since the badge is specifying master (default). Or do you think the link should show everything?

Reply:
> The image will use the default branch of the repo if not specified

Oh okay that does make sense .

I was pretty neutral on whether it should link to just results for master or for everything, how it is now is fine :)